acai-ios/CollectionViewCells/MenuTabCollectionViewCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  MenuTabCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Drew Dunne on 3/28/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+extension UIImage {
+
+    static func underlineImage(for text: String, andImage size: CGSize, font: UIFont) -> UIImage {
+
+        let textWidth = text.width(withConstrainedHeight: size.height, font: font)
+        let textHeight = text.height(withConstrainedWidth: textWidth, font: font)
+
+        print(size.height, textWidth)
```

Comment:
remove if unnecessary

Reply:
lol I was looking for that, couldn't find it. Thanks!